modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/ManagementToolbarPropsTransformer.js

```diff
@@ -0,0 +1,272 @@
+		}
+
+		processAction(action, editEntryURL);
+	};
+
+	const editTags = () => {
+		const searchContainer = Liferay.SearchContainer.get(
+			otherProps.searchContainerId
+		);
+
+		Liferay.componentReady(`${portletNamespace}EditTagsComponent`).then(
+			(editTagsComponent) => {
+				const bulkSelection =
+					searchContainer.select &&
+					searchContainer.select.get('bulkSelection');
+
+				const selectedFileEntries = searchContainer.select
+					.getAllSelectedElements()
+					.get('value');
+
+				editTagsComponent.open(
+					selectedFileEntries,
+					bulkSelection,
+					folderConfiguration.defaultParentFolderId
+				);
+			}
+		);
+	};
+
+	const move = () => {
+		const searchContainer = Liferay.SearchContainer.get(
+			otherProps.searchContainerId
+		);
+
+		let selectedItems = 0;
+
+		if (searchContainer.select) {
+			selectedItems = searchContainer.select
+				.getAllSelectedElements()
+				.filter(':enabled')
+				.size();
+		}
+
+		const dialogTitle =
+			selectedItems == 1
+				? Liferay.Language.get('select-destination-folder-for-x-item')
+				: Liferay.Language.get('select-destination-folder-for-x-items');
+
+		openSelectionModal({
+			height: '480px',
+			id: `${portletNamespace}selectFolder`,
+			onSelect(selectedItem) {
+				const newFolderId = selectedItem.folderid;
+
+				const form = document.getElementById(`${portletNamespace}fm2`);
+
+				if (!form) {
+					return;
+				}
+
+				form.setAttribute('action', editEntryURL);
+				form.setAttribute('enctype', 'multipart/form-data');
+				form.setAttribute('method', 'post');
+
+				form.querySelector(`#${portletNamespace}cmd`).setAttribute(
+					'value',
+					'move'
+				);
+				form.querySelector(
+					`#${portletNamespace}newFolderId`
+				).setAttribute('value', newFolderId);
+
+				submitForm(form, editEntryURL, false);
+			},
+			selectEventName: `${portletNamespace}selectFolder`,
+			size: 'lg',
+			title: Liferay.Util.sub(dialogTitle, [selectedItems]),
+			url: selectFolderURL,
+		});
+	};
+
+	return {
+		...otherProps,
+
+		// filterDropdownItems: filterDropdownItems.map((item) => {
+		// 	return {
+		// 		...item,
+		// 		items: item.items.map((childItem) => {
+		// 			return {
+		// 				...childItem,
+		// 				onClick() {
+		// 					const action = childItem.data?.action;
+
+		// 					if (action === 'openDocumentTypesSelector') {
+		// 						openSelectionModal({
+		// 							onSelect(selectedItem) {
+		// 								if (selectedItem) {
+		// 									const url = addParams(
+		// 										`${portletNamespace}fileEntryTypeId=${selectedItem.value}`,
+		// 										viewFileEntryTypeURL
+		// 									);
+		// 									navigate(url);
+		// 								}
+		// 							},
+		// 							selectEventName: `${portletNamespace}selectFileEntryType`,
+		// 							title: Liferay.Language.get(
+		// 								'select-document-type'
+		// 							),
+		// 							url: selectFileEntryTypeURL,
+		// 						});
+		// 					}
+		// 				},
+		// 			};
+		// 		}),
+		// 	};
+		// }),
```

Comment:
This place needs a bit of cleanup :broom: :slightly_smiling_face:

Reply:
:facepalm: that's what happens when I submit draft PRs in a hurry, will clean up